### PR TITLE
New docs

### PR DESCRIPTION
**Pull request by mbstacy**
Docs on the CU Library Read the Docs.

### CODE REVIEW

File: docs/products/documentation/readthedocs.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# CU Library Read the Docs
+
+ReadtheDocs is CU Boulder Libraries Core Tech & Apps approved method for documentation. 
+
+[Github Markdown Guides](https://guides.github.com/features/mastering-markdown/)
+
+[Read the Docs Documentation](https://docs.readthedocs.io/en/stable/index.html)
```
Comment (jwfuller) [inline docs/products/documentation/readthedocs.md:7]: Consider using a list

File: docs/products/documentation/readthedocs.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        $ . venv/bin/activate
+        $ cd docs/_build/html
+        $ python -m http.server
+        Serving HTTP on :: port 8000 (http://[::]:8000/) ...
+
+5. Open Browser [http://localhost:8000](http://localhost:8000)
+
+## Add new Documentation
+
+1. git checkout -b new_docs
```
Comment (jwfuller) [inline docs/products/documentation/readthedocs.md:50]: Only need to use `1.` for all numbers in ordered lists, it will figure out the rest of the numbers.

**Review by jwfuller — CHANGES_REQUESTED** [review]
Consider using code blocks to fence terminal commands.

**Comment by mbstacy** [comment]
@jwfuller I have read comments but for the demo will merge.

**Review by jwfuller — APPROVED** [review]

**Pull request merged**